### DISCUSSION TITLE
[Feature] "sticky" info display [existing Google Photos feature]

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#2998] [Feature Request]
### The feature

I would appreciate an option so that the "info" (when you click the "i" in image details view) is always on.   Potentially 1000's fewer clicks!

Thanks.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can hit the "i" key on your keyboard to toggle

**Reply by bcsteeve** [reply]
While that's a cool feature too... that replaces mouse clicks with key presses :)   An option to "pin" that info screen would be much welcome.

For me, I kind of think that should be the default :)  Its great info that I'd think people would want to see!

Not a huge priority.

**Reply by jrasm91** [reply]
I think persisting the open/closed state would be easy to add. Then it would already be open most of the time. Might be worth checking if Google does that or not.

**Reply by bcsteeve** [reply]
I see that sentiment a lot here... "check if Google does".  Is that really the goal, to be an exact copy of Google Photos?  Striving for better may be worthwhile :)

Afterall, Google Photos limits us to 15G and I don't see Immich rushing to introduce that limitation  lol

**Reply by bcsteeve** [reply]
Oh, and incidentally... yes, Google Photos does sticky the info display :)

**Reply by jrasm91** [reply]
I meant more of Google photos has really good design and this seems like a good idea - to persist the side panel open/close state. I had a feeling Google photos would have implemented something like this, so it was more it of curiosity than anything. Plus, it is easy to update the feature request to say - make this work like Google photos.

**Reply by bcsteeve** [reply]
And you were right :)

**Comment by alextran1502** [comment]
> I see that sentiment a lot here... "check if Google does". Is that really the goal, to be an exact copy of Google Photos? Striving for better may be worthwhile :)
> 
> Afterall, Google Photos limits us to 15G and I don't see Immich rushing to introduce that limitation lol

The goal is to provide the goodness without limitation :)

**Comment by martabal** [comment]
Hey do you want to persist even when you refreshed the page ?

**Reply by martabal** [reply]
It seems google photos persist the state even if the page is refreshed

**Reply by bcsteeve** [reply]
Me?  Yes.  Yes, that would be ideal, and I think logically expected by the user.  Thanks.

**Reply by martabal** [reply]
Fixed it in #3013, next release will have this feature :)

**Comment by samholton** [comment]
This feature request can be closed
